### PR TITLE
Feature 43

### PR DESCRIPTION
**Pull request by dhausk**
About page is fleshed out according to the Figma.  Sorry about the ton of commits....

### CODE REVIEW

File: src/components/Sponsors.tsx [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
+export function Sponsors(props: Props) {
   const image__css: CSSProperties = {
     margin: "0 auto",
     maxWidth: "250px",
     padding: 0,
   };
+  const showHeader = onAbout();
 
+  function onAbout() {
+    if(props.show == null) {
```
Comment (dviramontes) [inline src/components/Sponsors.tsx]: Although this code is technically a _correct_ way to use `==` operator for the intended of checking the type is `null`, I would prefer we use `===` everywhere due to the confusion that can arise in all of these different cases:

```js
const aNull = null

if (aNull == null) {
	console.log('always true')
}

=>  always true

if (aNull === null) {
	console.log('always true')
}
=>  always true

if (aNull) {
	console.log('always false')
}

=> undefined
```

File: src/components/AboutOrg.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          </Grid.Column>
+        </Grid.Row>
+        <Grid.Row>
+          <Grid.Column>
+            <Header as="h2">Our Sponsors</Header>
+            <p>
+              We couldn’t do any of this work without our sponsors.
+              If you’d like to learn more about how to support Code
+              for Denver,
+              <HashLink
```
Comment (dviramontes) [inline src/components/AboutOrg.tsx]: nice, didn't know about this!

**Comment by dviramontes** [comment]
bump @dhausk  ^ :)

File: src/components/Sponsors.tsx (hunk 1 above)
Comment (jberns) [reply src/components/Sponsors.tsx]: Fixed this

**Pull request merged**